bin/build_docker_images.sh

```diff
@@ -1,5 +1,13 @@
 #!/usr/bin/env bash
 
+# This script builds all the public Docker images that the Clipper project distributes
+# and pushes them to Docker hub.
+# Each image is pushed with the tag <image_name>:version
+# In addition, if the version is "develop", each image will also be
```

Comment:
I don't think this is accurate. We're always tagging images with a git hash, regardless of version. What's the reasoning behind only tagging of the version is "develop"?

Reply:
Yeah I wrote the comment before I wrote the script. Will update the wording to be consistent.